client/volume.go

```diff
@@ -149,19 +150,26 @@ func cmdVolumeDelete(c *cli.Context) {
 func doVolumeDelete(c *cli.Context) error {
 	var err error
 
-	name, err := getName(c, "", true)
+	names, err := getNames(c)
 	if err != nil {
 		return err
 	}
 
-	request := &api.VolumeDeleteRequest{
-		VolumeName:    name,
-		ReferenceOnly: c.Bool("reference"),
-	}
+	for _, name := range names {
+		request := &api.VolumeDeleteRequest{
+			VolumeName:    name,
+			ReferenceOnly: c.Bool("reference"),
+		}
 
-	url := "/volumes/"
+		url := "/volumes/"
+
+		err = sendRequestAndPrint("DELETE", url, request)
+		if err != nil {
+			fmt.Println("Error deleting " + name + ": " + err.Error())
```

Comment:
I would prefer reporting error to caller rather than stopping processing in the middle. I will change it. Thanks.